### ISSUE TITLE
v1.4.0 feedback

### ISSUE THREAD

**Issue by LennardF1989** [LennardF1989/NiohResolution#19]
Please leave feedback for v1.4.0 in this topic, mostly interested in the situations where people have to select a resolution other than 1920x1080 and still get the positive result they expect.

Please report:
- Version of the game (Steam/EGS)
- Patched resolution/scale
- Resolution selected in the launcher
- (Maximum) native desktop/monitor resolution

**Comment by Trokinos** [comment]
So far it seems to work well for me (EGS version) with these settings (for a lowspec system)
![001](https://user-images.githubusercontent.com/14024168/132988817-db7c00af-02ff-48ec-b3b9-5d79f013518a.png)

![002](https://user-images.githubusercontent.com/14024168/132988822-38c0c328-3b8c-4d97-98d7-d4dfb2ec158c.jpg)

(my monitor current resolution is 1920x1080)

**Comment by raynan222** [comment]
Version: EGS
Patched resolution/scale: 2560x1080, 1.0 scale
Monitor resolution: 2560x1080
GPU/Driver: 1050ti(4gb), driver 471.96

After patching, using 3840x2160 or 2560x1440 at game launcher allows the game to fill the screen(Without black bars), but performance is suboptimal.

**Comment by LennardF1989** [comment]
@raynan222 Could you add me on Discord (LennardF1989#3733)? I would like to test something to fix the performance degrading.

**Comment by HBbrody** [comment]
Version of the game (Steam/EGS): Steam
Patched resolution/scale: 3440x1440/1.0
(Maximum) native desktop/monitor resolution: 3440x1440
Seems to be working, but the HUD is not on the edges of the screen:
![2](https://user-images.githubusercontent.com/90416378/132992858-1aec85d5-b9dc-4e67-b0e4-b3a482c29620.jpg)

**Comment by LennardF1989** [comment]
@HBbrody
> Seems to be working, but the HUD is not on the edges of the screen

It never was in the older versions either, this is the best you are going to get I'm afraid.

Did you also select something other than 1920x1080 to get this result?

**Comment by HBbrody** [comment]
> Did you also select something other than 1920x1080 to get this result?

Yes, I have selected a different resolution and changed the display mode to full-screen. It doesn't work in borderless mode, which is fine for me.
There are also no performance problems, the game has a stable 60 frames per second.
My game settings:
![image](https://user-images.githubusercontent.com/90416378/132993189-347e2dd0-ffe6-4e38-a9ad-795a7f1de5fb.png)

**Comment by mhoogcarspel** [comment]
Version: EGS
Patched resolution/scale: 5120x1440, 1.0 scale
Monitor resolution: 5120x1440
GPU/Driver: 3080, driver 471.96

I selected the 4k option in the launcher

scaled down screenshot
![nioh32x9-scaled](https://user-images.githubusercontent.com/28956861/132993264-4037e59e-07a5-49fa-802e-3c1f9b617f52.png)

**Comment by ScratchMode** [comment]
Went 5 hours without a crash last night, so you definitely fixed that part of it 👍

**Comment by bazookaben** [comment]
So there are two problems, which one you get depends on whether you select 1920x1080 in the launcher, or one of the other resolutions.


- EGS
- 2560x1920 @ 1.0 (4:3)
- _Resolution selected in the launcher_: various, see below
- _(Maximum) native desktop/monitor resolution_:  N/A, I'm on a CRT. Output changes based on game settings


Results on a 4:3 monitor

**1920x1080:** 

Game will switch monitor to correct (patched) resolution of 2560x1920, but the viewport is letterboxed . The viewport is clearly rendering at the patched aspect ratio, except compressed inside the letterboxed area. 

So characters will look fat on a 4:3 monitor because the game will squish a 4:3 framebuffer into a 16:9 viewport

![4x3 squished](https://user-images.githubusercontent.com/36523399/132999301-3c3ef808-e5d2-453f-b8b1-b51f6d846802.jpg)

**All other selectable resolutions (in my case, 3840x2160):**

Game will correctly run at fullscreen with the correct aspect ratio. _However_ it will not have a matching pixel aspect ratio, because it is rendering at the resolution selected in launcher. So it will be either super sampling or down sampling, depending on your monitors resolution.

You can verify this by injecting a SpecialK64 DLL and looking at the frame buffer resolution in the GUI.

Also, you have no control over what output resolution the game switches your monitor to in fullscreen mode. The game forces will select whatever mode closest to what you have set in the launcher. So for example, if I select 2560x1440 in the launcher, the game will switch my monitor to 2304x1728, which is the closest sized resolution on my 4:3 monitor, and at 75hz since that's the only refresh rate I have available. And 60fps at 75hz is not pleasant, as you can imagine

For example, I patched the EXE to 2560x1920. Ran the launcher, selected 3840x2160.

Upon opening the game, it switched to 2880x2160 because that is the closest resolution I have to 3840x2160. Then I check Special K and see that the game is running a frame buffer of 3840x2160, outputting at 2880x2160.

![framebuffer fullscreen mismatch](https://user-images.githubusercontent.com/36523399/132999317-c31e6505-b54f-48fb-9e38-08b6ec39469c.jpg)

So in this case, I get super sampling on the horizontal axis. Looks good in this instance, but I wish I could control the framebuffer/rendering resolution to have more control over performance. I don't know if my GPU can handle rendering 4k60 in outdoor areas, for example.

And of course, if I had selected 2560x1440, it would switch the monitor's output to 2304x1728 which is up sampling on the horizontal, but downsamping on the vertical, which is a visual mess.

**Comment by bazookaben** [comment]
So, to sum up my previous post, 1080p in the launcher will get you a result that is _almost_ correct. 

Framebuffer is correct, fullscreen resolution is correct, and the game's output is rendered in the correct aspect ratio. The only problem is that the game is then squished back down to 16:9 with letterboxing or pillarboxing.

Maybe the correct solution is to just apply the resolution fix but _not_ the aspect ratio fix? It seems the game uses up the full screen unpatched, but it just stretches. So maybe just giving it the correct number of pixels will make it automatically run in the right aspect ratio.

**Comment by LennardF1989** [comment]
@bazookaben Thank you for that - this was my conclusion as well. My thought is the other way around: I should not touch the launcher resolution, but instead JUST modify the backbuffer. I'm pretty sure I tested the "don't apply AR, but do apply resolution"-fix, but it didn't seem to yield the expected results.

**Comment by bazookaben** [comment]
One issue with _only_ fixing aspect ratio is that you're limited to the handful of internal framebuffer resolutions provided by the launcher.

**Comment by qlQAQlp** [comment]
hmmm when i used my guardian ability?(idk whats that called) it stucks at a super cool post and crashes

i am using 1024 x 576 render 0.3

Using EGS 
launcher resolution 1920x1080 
Patched 1280x720 and render 0.3
Native PC max resolution 1920x1080
The game keep crashing after I killed someone and use guardian skill

The above situation didn’t happened when I was using the “nioh resolution” download from pull

Well I did applied the same setting as urs but when I kill the enemy suddenly appears behind the rock on to of the mountain of the village(I went all the way right when I arrived the island) the screen went black and the game crashes. Tho when I am playing offline I crashed after I killed the demon in the village ( I didn’t use guardian spirit I picked the fire dog either)

Fustrated lol

Tho when I went left it didn’t crashed instead I found that swimming is invented in 1520 so I died

There is a certain region that I can’t look at or else the game crashes????

I’ve certified and reinstalled the game I still crash when I look at that “region”

And it crashed again when I killed a demon

**Comment by qlQAQlp** [comment]
i am using 1024 x 576 render 0.3

**Comment by KVOfficial** [comment]
Version of the game -Epic Games Store
Patched resolution/scale-1366x768
Resolution selected in the launcher- 1980x1080
(Maximum) native desktop/monitor resolution- 1980x1080

The game wont save.
On an old save file it used to say failed to save to savefile0
the image below is on a new game/ save
![savefile](https://user-images.githubusercontent.com/90619816/133083246-2379575f-cd47-490d-86df-a155fcf90648.PNG)

**Comment by qlQAQlp** [comment]
Using EGS 
launcher resolution 1920x1080 
Patched 1280x720 and render 0.3
Native PC max resolution 1920x1080
The game keep crashing after I killed someone and use guardian skill

**Comment by qlQAQlp** [comment]
The above situation didn’t happened when I was using the “nioh resolution” download from pull

**Comment by Trokinos** [comment]
> 
> 
> Using EGS
> launcher resolution 1920x1080
> Patched 1280x720 and render 0.3
> Native PC max resolution 1920x1080
> The game keep crashing after I killed someone and use guardian skill

I use the same settings  and i've been clearing the burning village in the 1st mission, each time grinding the guardian spirit gauge to its max so i could take out each of the big demons (the ones that hide into the yokai smokes and do not respawn in the burning village if killed even when you go to a shrine).
And each time i have then used that cool living weapon skill to kill them easily (they stagger on each hits and take huge damage) as on normal condition i was always doing a mistake at some point and those huge demons killed me (the guardian spirit i use is the fire dog).  
Doing so i encountered no crash at all during the couple of hours i played there.

In case it can help you, 
On the Nioh launcher, i set the resolution to 1920x1080 and the Render Resolution to High (as they're both changed by the settings of the patch, so when i launch the game, the window will be as i want on 1280x720 and render scale of 0.3 )
In the Nioh system options, at the graphic setting i turned everythinb to Low and disabled everything that could be disabled (like the shadow).
Try this to see if there's one of the graphic option that is responsible of your crash.

**Comment by qlQAQlp** [comment]
> > Using EGS
> > launcher resolution 1920x1080
> > Patched 1280x720 and render 0.3
> > Native PC max resolution 1920x1080
> > The game keep crashing after I killed someone and use guardian skill
> 
> I use the same settings and i've been clearing the burning village in the 1st mission, each time grinding the guardian spirit gauge to its max so i could take out each of the big demons (the ones that hide into the yokai smokes and do not respawn in the burning village if killed even when you go to a shrine).
> And each time i have then used that cool living weapon skill to kill them easily (they stagger on each hits and take huge damage) as on normal condition i was always doing a mistake at some point and those huge demons killed me (the guardian spirit i use is the fire dog).
> Doing so i encountered no crash at all during the couple of hours i played there.
> 
> In case it can help you,
> On the Nioh launcher, i set the resolution to 1920x1080 and the Render Resolution to High (as they're both changed by the settings of the patch, so when i launch the game, the window will be as i want on 1280x720 and render scale of 0.3 )
> In the Nioh system options, at the graphic setting i turned everythinb to Low and disabled everything that could be disabled (like the shadow).
> Try this to see if there's one of the graphic option that is responsible of your crash.

Well I did applied the same setting as urs but when I kill the enemy suddenly appears behind the rock on to of the mountain of the village(I went all the way right when I arrived the island) the screen went black and the game crashes. Tho when I am playing offline I crashed after I killed the demon in the village ( I didn’t use guardian spirit I picked the fire dog either)

**Comment by qlQAQlp** [comment]
Fustrated lol

**Comment by qlQAQlp** [comment]
Tho when I went left it didn’t crashed instead I found that swimming is invented in 1520 so I died

**Comment by qlQAQlp** [comment]
There is a certain region that I can’t look at or else the game crashes????

**Comment by qlQAQlp** [comment]
I’ve certified and reinstalled the game I still crash when I look at that “region”

**Comment by Trokinos** [comment]
I don't know if it is related to your crash, but i notice that the epic store version has today an update to 1.24.02 (that i have not installed, i'm still on 1.24.01 as i am not in a hurry to update now that NiohResolution works)
I'm still playing on the version 1.24.01 i downloaded from them a couple of days ago (and so far no crashes in hours).

Can you check which version you are running, as it's possible the hex addresses have changed again in this new patch 1.24.02 and the current new 1.40 NiohResolution is not yet compatible.

**Comment by LennardF1989** [comment]
@qlQAQlp Please don't use this thread as a chatbox. Edit your previous message. I will delete/merge them.

@Trokinos The patch is harmless and should just work - it's built with forward compatibility in mind :)

For those with crashing issues, try running the game unpatched to verify.

**Comment by Trokinos** [comment]
Tested after the 1.24.02 update , patched the exe the same as i mentionned in my 1st report and it worked without problem so far for a hour of gameplay.

**Comment by qlQAQlp** [comment]
Hmmm somehow I still crashes on the mountain is there any way I can reverse the mod?( sorry fo spamming sooo much)

**Comment by qlQAQlp** [comment]
Hmmm somehow I still crashes on the mountain is there any way I can reverse the mod?( sorry fo spamming sooo much)

Ahhhh I found why I’ve turned on full screen optimisation sorry for bothering soo long
After I turned of full screen optimisation it ran at 60 FPS without crashing for 1 h

**Comment by jordillaurado** [comment]
Version of the game (Steam/EGS): Epic Games
Patched resolution/scale: 3440x1440/1.0
(Maximum) native desktop/monitor resolution: 3440x1440

![Screenshot 2021-09-14 103934](https://user-images.githubusercontent.com/34055916/133226162-e61db8f8-d4b0-464b-b50b-4b653dd13a7c.png)

Setting the resolution to 1080p causes the game to be pillarboxed.

![Screenshot (15)](https://user-images.githubusercontent.com/34055916/133226198-15b25ad1-a61a-4df2-b82d-61fc99dac7b0.png)

Choosing 2560x1440 instead of 1080p causes the game to play on borderless mode at this resolution. Choosing 4K causes the game to be cut vertically.

Edit: The problem was that I was using borderless instead of fullscreen. By switching to fullscreen and selecting 4K on the game launcher everything works fine.

**Comment by CDOccc** [comment]
![设置](https://user-images.githubusercontent.com/55381745/133276362-c8765c5f-c4c1-42a8-aeb4-ec711cf842ba.png)
![游戏启动项](https://user-images.githubusercontent.com/55381745/133276371-8c996b0c-7477-4a63-9ccc-de7b5b464a25.png)
![游戏界面](https://user-images.githubusercontent.com/55381745/133276380-6eb78bcd-5f44-4e64-93b0-6a8ae535fd97.png)

**Comment by llaltxll** [comment]
> 
> 
> Version: EGS
> Patched resolution/scale: 2560x1080, 1.0 scale
> Monitor resolution: 2560x1080
> GPU/Driver: 1050ti(4gb), driver 471.96
> 
> After patching, using 3840x2160 or 2560x1440 at game launcher allows the game to fill the screen(Without black bars), but performance is suboptimal.

Same here, different GPU: 3070ti, no performance issues, locked to 60fps.

**Comment by sevxz** [comment]
![image](https://user-images.githubusercontent.com/90839068/133602478-507ab1b4-15db-4988-bbba-c60b20439ab0.png)

![image](https://user-images.githubusercontent.com/90839068/133601393-0d80a363-e748-4806-8089-2869ee9fdd5c.png)

![image](https://user-images.githubusercontent.com/90839068/133601227-9ac32679-30dd-4a33-b71a-d6162f75facb.png)

Version: EGS
PatchedRes: 3440x1440
LauncherRes: 2560x1440
MonitorRes: 3440x1440

Everything works fine, you just can't run it in borderless mode otherwise you get pillar boxing and an awkward vertical stretch

**Comment by audiopedofil** [comment]
thank u 
![image](https://user-images.githubusercontent.com/90908374/133777114-8d0e4e3d-4631-40a0-853b-8e0793043637.png)

**Comment by audiopedofil** [comment]
it didn't help me

**Comment by maciejsojka** [comment]
Version: EGS
PatchedRes: 1920x1200
LauncherRes: 2560x1440
MonitorRes: 1920x1200@60Hz
GTX 1660 Super (driver ver. 471.96)

Works OK at: 2560x1440 - stable 60 FPS, 3840x2160 - to low FPS (40-50)
After I choose 1920x1080 in launcher screen isn't fine.

The game requires a lot of practice from me because I play mouse and keyboard :-)

**Comment by Ted-Barrett** [comment]
Version: EGS
Monitor res & patched res: 3440x1440
Launcher res: 2560x1440 (1920x1080 gives pillarbox)

Works OK, however horizontal resolution is still 2560 (although when stretched to full screen, the FOV is correct, as well as the hud, but it's a bit blurry).

FPS pretty much locked at 60 with occasional stutters (2070 SUPER)

**Comment by rajder656** [comment]
Version of the game -Epic Games Store
Patched resolution/scale-1920x1200
Resolution selected in the launcher- 1980x1080
(Maximum) native desktop/monitor resolution- 1980x1200

The game/ launcher is in japanese
![image](https://user-images.githubusercontent.com/55133382/133905653-99975e03-9fb6-4e58-9e82-4d3ee8e7550a.png)
![image](https://user-images.githubusercontent.com/55133382/133905679-7dc51e09-f46e-4284-9227-dc9474541dca.png)

**Comment by LennardF1989** [comment]
@rajder656 This has nothing to do with the patch. You have to launch the game through Epic Games launcher, not directly. Otherwise what you get is the result.

**Comment by mk-ultron** [comment]
I'm having issues getting it to work on 3840 x 1600. Anyone figure that resolution out yet?

**Comment by Jade-mist** [comment]
Version of the game: EGS
Patched resolution/scale: 1920x1200 1.0
Resolution selected in the launcher: 1920x1080
(Maximum) native desktop/monitor resolution: 1920x1200
I am getting black stripes on the bottom and top of the screen when the resolution selected in the launcher is 1920x1080. It could be because the EGS version of the game is 1.24.02 and not 1.24.01? It looks good when choosing 2560x1440. On 3840x2160 it also looks good, but the framerate drops from ~60 to ~35. There is noticeable stuttering every few seconds on all resolutions. 

When the resolution selected in the launcher is 1920x1080:
![image](https://user-images.githubusercontent.com/50962756/134174981-a1472914-3f16-4d26-8e02-21fcf68f8f9a.png)

Edit: I don't think that the stuttering is because of the patch.
Edit 2: The stuttering disappeared (not sure why). 2560x1440 and 3840x2160 is too much for my GPU to handle, so I tried 2048x1152. The FPS is a stable 60 and the resolution seems to be right.

**Comment by bazookaben** [comment]
@Jade-mist Yeah the problem is the resolution tool is causing the game to switch to letterboxing. So the only way to prevent that is to _not_ use a custom resolution, i.e. switching to 2560x1440 (which sounds like it's a bit too intense for your GPU), but still use the patch for the aspect ratio correction.

So this tool would be perfect if it could allow switching to a custom resolution without then switching _on_ letterboxing. But it seems there's something hidden in the exe causing this trigger.

**Comment by Jade-mist** [comment]
@bazookaben Oh, I see. Do you think I would have the same problem on Steam?

**Comment by bazookaben** [comment]
I think the Steam version is basically identical

You might be able to use Special K to force a resolution, but I haven't figured out how to use it without crashing the game.

**Comment by PandaThePanda** [comment]
I got the game Nioh from the Epic Games Store and I got this mod since it would allow me to play the game at higher fps and actually run more smoothly. Most of the instructions I see here are for the Nioh game on steam not from the Epic games launcher. I heard this release can support both the steam and epic games version. I've been trying but nothing has seem to be working. Im not sure what I'm doing wrong and the only instructions I see are for the steam version of the game

Can someone give me a step-by-step on how to use this resolution mod for Nioh from Epic Games?

**Comment by LennardF1989** [comment]
When I have some spare time I'll dive into the dissembler again to find some new offsets that might counter this.

**Comment by PandaThePanda** [comment]
Which folder should I put the res mod in? I placed it in the KoeiTecmo folder, nothing happened. I tried placing it in the game folder titled NIOH, nothing happened. I don't know what I'm doing wrong. Should I be leaving the game open while I do this? I am opening the game from epic games btw. Im not really sure what I'm doing wrong. Can someone help me out with this?

The error message I'm getting back from the res mod says that it could not find Nioh.exe

**Comment by Jade-mist** [comment]
@PandaThePanda 
1) You should put NiohResolution.exe in the same folder nioh.exe is. For me that is C:\Program Files\Epic Games\Nioh. 
2) Start the NiohResolution.exe and follow the steps (they are pretty simple). Remember to close the game before doing this (just in case).
3) You should start the game from the Epic Games launcher. Remember to set the 'Rendering Resolution' to 'High', the 'Display Mode' to 'Full Screen' and the 'Screen Resolution' to '1920x1080' in the Nioh launcher (the launcher that appears after you start the game from Epic Games).
4) That should be it!
**Note**: For some resolutions setting the Nioh launcher to '1920x1080' does not work very well. If there are problems, you should change the Nioh launcher resolution to a different one that is close to your monitor's native resolution. You can try reading above to see what problems people have experienced and what possible solutions they have found, it may help.

This is what the Nioh launcher's 'Settings' tab looks like:
![image](https://user-images.githubusercontent.com/50962756/134288880-00c37548-9715-495b-a261-64ebc7fecfa1.png)

**Comment by PandaThePanda** [comment]
@Jade-mist, 

THANK YOU SO MUCHHHHHHH
YOUR TIPS AND STEP BY STEP WITH PICS ACTAULLY ENDED UP WORKING, THANK YOU FOR MAKING MY DAY!!!
You have no idea how long I've been trying lmao.
thank you for helping me out with this

**Comment by duahsjbsjbdqu27** [comment]
Hello,

Maybe I am making a mistake, but there is no executable file in the github repository (as of today). Could you help me out?

**Comment by Jade-mist** [comment]
@duahsjbsjbdqu27 
I am not completely sure that this is what you are asking, but I downloaded the .exe from 'Releases' .
![image](https://user-images.githubusercontent.com/50962756/134518327-34fcc2fb-9e8f-4b0a-8641-e842b56e644a.png)

Here is the direct link: https://github.com/LennardF1989/NiohResolution/releases/tag/v1.4.0

**Comment by duahsjbsjbdqu27** [comment]
> 
> 
> @duahsjbsjbdqu27
> I am not completely sure that this is what you are asking, but I downloaded the .exe from 'Releases' .
> ![image](https://user-images.githubusercontent.com/50962756/134518327-34fcc2fb-9e8f-4b0a-8641-e842b56e644a.png)
> 
> Here is the direct link: https://github.com/LennardF1989/NiohResolution/releases/tag/v1.4.0

ahhh silly me, thank you! That was exactly what I was asking for!

**Comment by KrechetSt** [comment]
Version: EGS
Patched resolution/scale: 1280x960 and 1280x1024, 1.0 scale
Monitor resolution: 1280x1024
GPU/Driver: GTX1060(6gb), driver 472.12

When I input resolution more than 1280x720 the image becomes compressed from top to bottom (horizontal stretch) with the HUD being not near the edges of the screen. The farer resolution is from 1280x720, the more compressed becomes the image. The problem persists in both full screen and windowed modes, though the previous version worked fine for both modes on v1.21.06.
Is there any way to resolve this problem?
![NiohBug03](https://user-images.githubusercontent.com/57035175/134533395-dc87493b-0626-4e51-b122-219caca2ac2b.jpg)
![NiohBug04](https://user-images.githubusercontent.com/57035175/134533408-e7da22c2-5bf5-402e-8792-52fcbf013be9.jpg)

**Comment by bazookaben** [comment]
@KrechetSt you should set the patcher to 1280x1024, with a 0.5 resolution scale. Then set 2560x1440 or 3840x2160 in your launcher. 

If I understand correctly, this will make the game run at 1280x720 or 1920x1080, respectively, but the aspect ratio would be correct (internal pixels won't be square anymore but your picture should have the right aspect ratio).

But I'm not 100% sure if the 0.5 resolution scale sticks when you change to 1440p or 2160p in the launcher. If it doesn't, and you have bad performance, you can try setting "rendering resolution" to "low" or "medium" in the launcher.

Not a perfect solution, you'll get some blurry scaling if you land below your native resolution, but it'll let you play the game fullscreen anyway.

**Comment by bazookaben** [comment]
@KrechetSt you should set the patcher to 1280x1024, with a 0.5 resolution scale. Then set 2560x1440 or 3840x2160 in your launcher. 

If I understand correctly, this will make the game run at 1280x720 or 1920x1080, respectively, but the aspect ratio would be correct (internal pixels won't be square anymore but your picture should have the right aspect ratio).

But I'm not 100% sure if the 0.5 resolution scale sticks when you change to 1440p or 2160p in the launcher. If it doesn't, and you have bad performance, you can try setting "rendering resolution" to "low" or "medium" in the launcher.

Not a perfect solution, you'll get some blurry scaling if you land below your native resolution, but it'll let you play the game fullscreen anyway.

**Comment by KrechetSt** [comment]
@bazookaben that doesn't work.
When I chose 1920x1080 the stretch persists. And when I select the resolution that exceeds the one of my display, the game window doesn't fir to the monitor. And I need 1280x960 in the windowed mod. I can easily get 1280x720 without any mods, but I needn't it.

Does anybody know any other solution? And the source of the problem?
And what does render resolution scale do? Just increases performance by deteriorating the quality of the image?

**Comment by Trokinos** [comment]
@KrechetSt 
problem i think related to the stretching is that 1280x960 is not a 16/9 resolution (while 1280x720 is) , it seems that every custom resolution that is not a 16/9 one can result in some stretching. Probably it's the way the game was coded and not really really optimised for PC usage.    
The resolution scale is not only for increasing performance by lowering the image quality, it can be used for the opposite, upscaling the visuals (if you use a resolution scale superior to 1)  
  
There has been a new update on the EGS version, 1.24.3 :  
https://updatecrazy.com/nioh-update-1-24-3-patch-notes-for-pc-sep-27-2021/
  
Happy to report that NiohResolution still works on this new version

**Comment by mjadev360** [comment]
Version of the game: EGS
Patched resolution/scale: 2560 x 1080, 1.0
Resolution selected in the launcher: 4k 
(Maximum) native desktop/monitor resolution: 3840 x 2160
GPU: RX 6800

works, but ONLY if it's set to fullscreen. Took me a dozen or more attempts to get it working because I was not aware that it would not work in borderless mode.  This should be added to the description/instructions because it does not say that anywhere.

setting the launcher to 1920 x 1080 would cause pillarboxes and the content to be smushed

Also, sometimes running NiohResolution.exe would cause the launcher to be in Japanese

**Comment by r0n1n999** [comment]
Version of the game (Steam/EGS): Steam
Patched resolution/scale: 2560*1080
Resolution selected in the launcher: 1920*1080
![Без имени-1](https://user-images.githubusercontent.com/92083312/136350392-904a2b42-2358-4c7d-8fe3-8fc8a7619c00.jpg)
![123](https://user-images.githubusercontent.com/92083312/136350435-fd08b561-8a34-41e5-b569-474165b938c5.jpg)
![1234](https://user-images.githubusercontent.com/92083312/136350458-a52095b8-a2bc-4e71-b46b-c0524c4e20f2.jpg)

(Maximum) native desktop/monitor resolution: 2560*1080
Version game: 1.24.3
Version fix: 1.4.0
Version driver: 472.12 (GTX 1060)

black stripes go away only if you put 2560 x 1440

**Comment by Grimjoveth** [comment]
Version: Steam (1.24.3)
Patched resolution/scale: 3440x1440/1.0
Resolution Selected - Tried all 
(Maximum) native desktop/monitor resolution: 3440x1440

Installing the patch causes the game to crash at launch. Will not work under any launcher resolution or display mode change.
Game will only launch again successfully after Steam file integrity check replaces 1 file.
Have used your previous patch, v1.3.0 and did not run into this problem.
Reinstalling the game does not fix issue.
![Nioh](https://user-images.githubusercontent.com/93153113/138749257-155e54ca-088d-449a-ad9f-7ed4d28c12e3.jpg)
![NiohResolution](https://user-images.githubusercontent.com/93153113/138749908-a1311687-fb87-4ac2-9646-db2d7d707c6c.jpg)

**Comment by milanesanapolitana** [comment]
Version of the game: 1.24.03 Steam
Patched resolution/scale: 2560x1080
Resolution selected in the launcher: 1920x1080, then 2560x1440, then 3840x2160
(Maximum) native desktop/monitor resolution: 2560x1080

I think I've tried every launcher res - patched res combination but the game always looks like I'm playing in 16:9 1080p. No launcher resolution looks any different unless I select a low resolution, obviously, the game shows me fewer pixels. Still, always 16:9. Maybe I'm missing something obvious but I can't get it to work on my 21:9 monitor

![123456](https://user-images.githubusercontent.com/89754924/140619556-ef4a12a9-7090-43b2-82f7-817167943b04.png)
![1234562](https://user-images.githubusercontent.com/89754924/140619557-551fa9cb-960c-4c73-8094-18e95726fcd1.png)

**Comment by Name-Searcher** [comment]
Version of the game: 1.24.03 Steam
Patched resolution/scale: 2560x1080 1.0 FMV patch = no
Resolution selected in the launcher: 1920x1080, 2560x1440, 3840x2160

my native monitor resolution is 2560x1080 but it also accepts 3840x2160 and downcales the resolution to the panel size itself so picking a higher resolution just runs it bigger but letterboxed (steam overlay also shows it running as 16/9 in these modes only on 1920x1080 does it properly align to the sides of my screen)

running 1920x1080 does run the rendering in 2560x1080 but it warps the image to letterboxed 16/9 making it looked stretched out and weird and all the loading screen show up looking like 4/3 resolutions instead of the 16/9 they normally are

afterwards i tried playing on the backup exe but that had a weird loading thing making it take forever so i had to verify my files with steam before i could play again
![Screenshot (21)](https://user-images.githubusercontent.com/70028020/144931262-cef7e8d6-ea17-4fdc-9117-0b694381d5d7.png)
![Screenshot (22)](https://user-images.githubusercontent.com/70028020/144931264-c1a6b9a9-91e6-4910-a7f3-2f487ab40025.png)
![Screenshot (23)](https://user-images.githubusercontent.com/70028020/144931267-5d09a5cc-10fc-414c-9b86-05d082baa086.png)
dont mind the backup had to redo for the screenshot because windows key closed the window (restoring this before didn't seem to work only verify files through steam)
![Capture](https://user-images.githubusercontent.com/70028020/144931599-85693a7d-f00c-4529-87eb-12d49dcc4f25.PNG)
![1](https://user-images.githubusercontent.com/70028020/144931886-77b6660c-62de-42df-8ddf-d11eaf5f991b.PNG)

**Comment by HarshaKarapureddy** [comment]
Hi, so when I use this on my game folder it keeps on saying that nioh.exe is denied access. I feel like I missed something and I would like to know what the problem here is.

![error](https://user-images.githubusercontent.com/99938620/154611474-ddd7439a-0dd9-4020-9252-66176aee2a97.PNG)

**Comment by LennardF1989** [comment]
> Hi, so when I use this on my game folder it keeps on saying that nioh.exe is denied access. I feel like I missed something and I would like to know what the problem here is.
> 
> ![error](https://user-images.githubusercontent.com/99938620/154611474-ddd7439a-0dd9-4020-9252-66176aee2a97.PNG)

Have you tried running it as Administrator? Could be either your game path is in a protected location in Windows (it happens), or there are non-latin characters in the game path.

**Comment by HarshaKarapureddy** [comment]
> 

Thanks, man, that helped. You're amazing for still taking care of this thread even though it's been nearly a year. Thanks again and have a good day.

**Comment by DGPhil** [comment]
> Version of the game (Steam/EGS): Steam Patched resolution/scale: 3440x1440/1.0 (Maximum) native desktop/monitor resolution: 3440x1440 Seems to be working, but the HUD is not on the edges of the screen: ![2](https://user-images.githubusercontent.com/90416378/132992858-1aec85d5-b9dc-4e67-b0e4-b3a482c29620.jpg)

How did you get the screenshot with properly aspect? Using steam screenshot key?
My monitor resoulution and settings all same with you, and I can see the correct aspect 21:9 in the monitor, but the screenshots is 16:9 (using steam screenshot key)
Even though I tried 1080p / 4k fullscreen, the screenshots still 16:9
I can confirm windowed / borderless is not working

![20220428175748_1](https://user-images.githubusercontent.com/6955282/165734567-615cbf26-69cc-4e6e-9976-bbf5d05266bf.jpg)

**Comment by POGMANNERS** [comment]
Hello.
I'm running the latest EGS release.
My patched resolution: 2560x1080 scale=1.0
I tried 4K, 2560x1440 and 1920x1080 but the results are the same. (Except that the scaling changes with bigger res)
I tried fullscreen, borderless and windowed mode but the results are the same.
The game squeezes the 2560x1080 resolution into a 16:9 window/box.
The game used to work before a patch and with v1.3.0 (if I remember correctly) but since v1.3.0 can't work I had this issue.
Not sure if I posted this to the right place, so feel free to point me to the right forum.
![Nioh 1 24 07 2022  05  21  10_30_53](https://user-images.githubusercontent.com/101746913/169643471-1ebb9426-1362-4161-8a9a-aed818e09492.png)
